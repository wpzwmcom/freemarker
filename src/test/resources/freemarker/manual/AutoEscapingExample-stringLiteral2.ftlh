<#ftl output_format="HTML"><#-- Needed for the Eclipse plugin -->

<#-- Markup output value created by escaping plain text: -->
<#assign mo1 = "Foo & bar"?esc>
<#-- Markup output value created outherwise: -->
<#assign mo2 = "<p>Foo"?no_esc>

${"${mo1} baz"}
<#attempt>${"${mo2} baz"}<#recover>Failed</#attempt>