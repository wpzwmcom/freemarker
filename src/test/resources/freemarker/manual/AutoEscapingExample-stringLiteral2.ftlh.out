
Foo &amp; bar baz
Failed